Libraries/src/net/relinc/libraries/unitTests/StrainGaugeTest.java

```diff
@@ -14,9 +14,10 @@
 public class StrainGaugeTest extends BaseTest{
 	@Test
 	public void testLoadStrainGauge(){
+
 		StrainGauge sg = new StrainGauge("SG", 1.0, 2.0, 3.0, 4.0, 5.0);
 		String fileString = sg.stringForFile();
-		File file = new File(TestingSettings.testingOutputLocation + "/" + sg.genericName + ".txt");
+		File file = new File(TestingSettings.testingOutputLocation + "/" + sg.genericName + ".json");
```

Comment:
If we were writing really good tests, we'd have the legacy format around to test and make sure it still works. But I dunno if it's worth it